### ISSUE TITLE
Two questions related to JPEG-XL functionality

### ISSUE THREAD

**Issue by ZPNRG** [tannerhelland/PhotoDemon#579]
1) Is there a way from within PhotoDemon 2024.7 to see what version of the libjxl library is installed and being used by PhotoDemon?  I'm curious since they have just a couple of weeks ago updated all of the v0.7, v0.8, v0.9, and v0.10 series of builds to v0.10.3, v0.9.3, v0.8.3, and v0.7.1.  I would think that you would probably want to be using the v0.9.3 or v0.10.3 build, but I don't know what version PhotoDemon is downloading.

2) Where are the libjxl library file(s) being download/stored to?  I have chosen since the beginning in 2022 when I started using PhotoDemon to use the portable version.  That might change, but that is what I am still using now.  It appears the libjxl library files are being downloaded to the App\PhotoDemon\Plugins folder.  I see cjxl.exe, djxl.exe, and jxlinfo.exe.  I have not investigated enough to know if you need to download and use any other libjxl files.  I don't see any.  From the file sizes, it looks like the current libjxl library files that PhotoDemon downloaded are not version v0.10.3 or v0.9.3, but I haven't investigated further.

**Comment by tannerhelland** [comment]
Thank you for the question, @ZPNRG.

You can see the current libjxl version PhotoDemon is using in the `Tools > Third-Party Libraries` screen.

The PD stable release from earlier this week shipped with v0.10.0 of libjxl.  I have just posted an update that supports the latest 0.10.3 libjxl .  PD should offer to download the update the next time you interact with a JPEG-XL file, if your PD update options allow (`Tools > Options > Updates`).

**Comment by tannerhelland** [comment]
Oops, I missed part of your second question.  I'm sorry!

> Where are the libjxl library file(s) being download/stored to? 

As you discovered, all third-party libraries (including libjxl) are kept in PD's `App\PhotoDemon\Plugins` folder. 

> I see cjxl.exe, djxl.exe, and jxlinfo.exe. 

These, plus the libjxl license file, are the only files PD downloads and configures.  (These files are taken byte-for-byte from the associated official libjxl release, and you can overwrite them with your own copies at your leisure as new versions become available.)

**Comment by ZPNRG** [comment]
I poked around in PD but somehow overlooked the Tools > Third-Party Libraries screen.  I figured you had to have something like that somewhere.  Sorry I overlooked that.

As far as the 2nd question I had, yes, I had also seen the libjxl license text file.  I was wondering if I could simply replace the cjxl.exe, djxl.exe, and jxlinfo.exe files as new versions of jxl-x64-windows-static.zip were released.  Thank you for letting me know that should be okay.  Granted, if just interacting with a JPEG-XL file will force PD to do it, that would be easier/faster than me doing it manually.
